set.mm

```diff
@@ -147154,6 +147148,398 @@ of an open range of nonnegative integers (of length equal to the length of
+      cpfx ccatcl wrdf 3syl ccatlen simp1 fzass4 biimpri simpld 3adant1 swrdlen
+      pfxlen oveq12d cc cz elfzelz ad2antrl 3impb ad2antll pncan3d oveq2d mpbid
+      ffn fneq2d pfxfn 3adant2 cv wo 3ad2ant2 fzospliti eleq2d biimpar ccatval1
+      simpr ccatval2 anim1i ancomd fzosubel syl wb subidd eqcomd oveq1d eqeltrd
+      mpbird swrdfv syldan elfzoelz adantl npcand fveq2d 3eqtrd jaodan eqfnfvd
+      pfxfv simpl3 eqtr4d ) BAUBZEZCFDGHEZDFBISZGHZEZUCZUAFDJHZBCULHZBCDUDUEHZU
+      FHZBDULHZXOXSFXSISZJHZUGZXSXPUGXOXSXIEZYBAXSUHYCXOXQXIEZXRXIEZYDXJXKYEXNA
+      BCUIUJZXJXKYFXNABCDUKUJZAXQXRUMKAXSUNYBAXSVNUOXOYBXPXSXOYADFJXOYAXQISZXRI
+      SZLHZDXOYEYFYAYKMYGYHAXQXRUPKXOYKCDCNHZLHDXOYICYJYLLXOXJCXMEZYICMXJXKXNUQ
+      ZXKXNYMXJXKXNOZYMDCXLGHEZYMYPOYOFCDXLURUSUTVAZABCVCKZABCDVBVDXOCDXJXKXNCV
+      EEZXJYOOZCXKCVFEZXJXNCFDVGZVHPVIZXJXKXNDVEEYTDXNDVFEXJXKDFXLVGVJPVIVKQZQV
+      LVOVMXJXNXTXPUGXKBDAVPVQXOUAVRZXPEZOZUUEXSSZUUEBSZUUEXTSZXOUUFUUEFCJHZEZU
+      UECDJHZEZVSZUUHUUIMZUUGUUFUUAUUOXOUUFWEZXOUUAUUFXKXJUUAXNUUBVTZRUUEFDCWAK
+      XOUULUUPUUNXOUULOZUUHUUEXQSZUUIUUSYEYFUUEFYIJHZEZUUHUUTMXOYEUULYGRXOYFUUL
+      YHRXOUVBUULXOUVAUUKUUEXOYICFJYRVLWBWCAXQXRUUEWDTUUSXJYMUULUUTUUIMXOXJUULY
+      NRXOYMUULYQRXOUULWEUUECABXFTQXOUUNOZUUHUUEYINHZXRSZUVDCLHZBSZUUIUVCYEYFUU
+      EYIYKJHZEZUUHUVEMXOYEUUNYGRXOYFUUNYHRXOUVIUUNXOUVHUUMUUEXOYICYKDJYRUUDVDW
+      BWCAXQXRUUEWFTXOUUNUVDFYLJHZEUVEUVGMUVCUVDUUECNHZUVJXOUVDUVKMUUNXOYICUUEN
+      YRVLRZUVCUVKUVJEZUVKCCNHZYLJHZEZUVCUUNUUAOUVPUVCUUAUUNXOUUAUUNUURWGWHUUEC
+      DCWIWJXOUVMUVPWKUUNXOUVJUVOUVKXOFUVNYLJXKXJFUVNMXNXKUVNFXKCXKCUUBPWLWMVTW
+      NWBRWPWOABCDUVDWQWRUVCUVFUUEBUVCUVFUVKCLHUUEUVCUVDUVKCLUVLWNUVCUUECUUNUUE
+      VEEXOUUNUUEUUECDWSPWTXOYSUUNUUCRXAQXBXCXDWRUUGXJXNUUFUUJUUIMXOXJUUFYNRXJX
+      KXNUUFXGUUQUUEDABXFTXHXE $.
+  $}
+
+  ${
+    $d k B $.  $d k S $.  $d k T $.
+    $( Recover the left half of a concatenated word.  (Contributed by Mario
+       Carneiro, 27-Sep-2015.)  (Revised by AV, 6-May-2020.) $)
+    pfxccat1 $p |- ( ( S e. Word B /\ T e. Word B )
+                      -> ( ( S ++ T ) prefix ( # ` S ) ) = S ) $=
+      ( vk wcel wa chash cfv cc0 cfzo cfz wceq cn0 lencl adantr syl2anc wfn syl
+      co eqtrd cword cconcat cpfx cres ccatcl caddc anim12i sylib elfz0add sylc
+      nn0fz0 ccatlen oveq2d eleqtrrd pfxres wss ccatvalfn cz nn0zd uzid uzaddcl
+      cuz syl2an fzoss2 fnssres wrdfn cv fvres adantl ccatval1 3expa eqfnfvd )
+      BAUAZEZCVMEZFZBCUBSZBGHZUCSZVQIVRJSZUDZBVPVQVMEVRIVQGHZKSZEVSWALABCUEVPVR
+      IVRCGHZUFSZKSZWCVPVRMEZWDMEZFVRIVRKSEZVRWFEVNWGVOWHABNZACNZUGVNWIVOVNWGWI
+      WJVRUKUHOVRWDVRUIUJVPWBWEIKABCULUMUNAVQVRUOPVPDVTWABVPVQIWEJSZQVTWLUPZWAV
+      TQBCAUQVPWEVRVBHZEZWMVNVRWNEZWHWOVOVNVRUREWPVNVRWJUSVRUTRWKWDVRVRVAVCVRIW
+      EVDRWLVTVQVEPVNBVTQVOABVFOVPDVGZVTEZFWQWAHZWQVQHZWQBHZWRWSWTLVPWQVTVQVHVI
+      VNVOWRWTXALABCWQVJVKTVLT $.
+  $}
+
+  $( A prefix of length 1.  (Contributed by AV, 15-May-2020.) $)
```

Comment:
Not sure I'm using the correct words, but maybe:
The prefix of length one of a given nonempty word expressed as a single character word.

Reply:
Done (The prefix of length one of a nonempty word expressed as a singleton word.)